benchmarks/benchmarks.py

```diff
@@ -13,12 +13,27 @@
     ModalLocation,
     Flows,
     TotalLocationEvents,
+    subscriber_location_cluster,
+    EventScore,
+    MeaningfulLocations,
+    MeaningfulLocationsAggregate,
+    MeaningfulLocationsOD,
 )
 from .utils import get_env_var
 from .config import FLOWDB_CONFIGS, FLOWDB_CONFIG_PARAM_NAMES
 from .flowdb_config import FlowDBConfig
 
 
+def make_params(params_dict):
+    """
+    Return a list of params and a list of their names, including the FlowDB
```

Comment:
This wasn't quite clear to me at first and I had to look at the usage examples to understand the purpose of this helper function.

How about: _"Takes a list of benchmark-specific parameters in params_dict and returns the full list of parameters and their names, including the FlowDB params defined in config.py"_

I think it would also help to have a simple usage example and output (and/or add the `Parameters` and `Returns` sections that we have in flowmachine docstrings?).